examples/grpc/helloworld_client/main.go

```diff
@@ -20,7 +20,7 @@ import (
 	"os"
 	"time"
 
-	"go.opencensus.io/examples/grpc/exporter"
+	"go.opencensus.io/examples/exporter"
```

Comment:
Wait, does this code compile?

Reply:
Yes, I moved the exporter package.

Reply:
Oh I see, sorry for the false alarm
<img width="975" alt="screen shot 2018-02-22 at 4 37 58 pm" src="https://user-images.githubusercontent.com/4898263/36572106-c7ad442e-17ee-11e8-8071-42b6cceb1331.png">